README.md

```diff
@@ -0,0 +1,177 @@
+
+
+```bash
+docker pull jupyter/base-notebook
+```
+```bash
+docker pull python
+```
+```bash
+docker pull postgres
+```
+
+
+
+- Then, type the following command to initialize all the containers that the project includes:
+
+```bash
+docker-compose up
+```
+
+## Opening Apache Airflow server
+
+If you want to go directly to Apache Airflow server, where the DAG is scheduled, please open your browser and type in a new tab:
+
+```bash
+localhost:8080
+```
+
+This command will open the web page that is running Apache Airflow GUI. You will need to insert a login and password to login into the server. 
+  
+
+## Opening Jupyter Notebook server
+
+If you want to go directly to the Jupyter Notebook analyzes, please open your browser and type in a new tab:
+
+```bash
+localhost:8888
+```
+
+This command will open the container's content that includes csv_analyze.ipynb file with the building permissions table analyze. In order to check the results of the analyze in the browser tab, please click on the "Run" tab of the displayed page and choose the suboption of "Run All Cells"
+
+![IMG IPYNB1](https://github.com/JakBiel/README_img/blob/main/jupyter_scr2.PNG)
+
+The step above will generate plots with various analysis of the data.
+
+## Schema of the project - directory tree
+
+```bash
+lab3
+│   .gitignore # This file excludes from GIT all the files needed locally only like e.g. virtual environmen etc.
+│   docker-compose.yaml # The file to simultaneous management of all the docker container used in the project
+│   README.md # The file with project's documentation
+│
+├───airflow_dockerfile # The folder for the Dockerfile of all the Apache Airflow containers
+│       Dockerfile # Dockerfile used for creation of all the Apache Airflow containers
+│
+├───dags # The folder for DAG's code pieces which are programmed in Python language
+│       aggregates_python.py # The main Python file of the DAG presented in the project
+│       aggregates_python_helpers.py # The Python file including the helpers of the main Python file "aggregates_python.py" 
+│   
+│
+└───zip_data_jupyter_notebook # The folder for the JupyterNotebook docker container designed for JupyterNotebook analyzes
+    │   requirements.txt # List of programs to be pre-installed in order to correctly execute the whole Jupyter application
+    │   Dockerfile # Dockerfile used for the creation of the JupyterNotebook docker container
+    │   config.yaml # File used for setting the configuration of the connection between JupyterNotebook container and BigQuery database
+    │
+    └───jpdata
+            csv_analyze.ipynb # The file with analyzes of data informing about building permissions in Poland
+
+```
+
+## Example of email sent by DAG
+
+![IMG MAIL_EXMPL](https://github.com/JakBiel/README_img/blob/main/email_example.PNG)
+
+## Example of email attachment with Great Expectations validation report 
+
```

Comment:
Did you open downloaded html ? Please do it the results are much prettier and better formatted. After that please make the screen and put here

Reply:
I will take a look at the report downloaded on Desktop and check it.